support/ebpf/types.h

```diff
@@ -515,7 +515,11 @@ typedef struct __attribute__((packed)) ApmCorrelationBuf {
 // Container for a stack trace
 typedef struct Trace {
   // The process ID
+  // NOTE: Confusingly, this is what Linux calls "tgid"
```

Comment:
Maybe it is worth to mention, that `pid` references here the user space wording and differentiate it from the kernel space wording and its return from the eBPF helper. `Linux` can be an overloaded term.